phoenix-core/src/main/java/org/apache/phoenix/schema/stats/StatisticsWriter.java

```diff
@@ -31,6 +31,8 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import org.apache.commons.logging.Log;
```

Comment:
let's use `org.slf4j` here instead of `org.apache.commons` to make consistent.
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

Reply:
Resolved